packages/roosterjs-content-model-dom/lib/domToModel/context/defaultProcessors.ts

```diff
@@ -55,6 +55,7 @@ export const defaultProcessorMap: ElementProcessorMap = {
     table: tableProcessor,
     u: knownElementProcessor,
     ul: listProcessor,
+    del: knownElementProcessor,
```

Comment:
please do alphabet order

Reply:
Sounds good. Addressed this in latest commit.